integ-test/src/test/java/org/opensearch/sql/legacy/AggregationIT.java

```diff
@@ -241,31 +248,34 @@ private void assertResultForGroupByTest(JSONObject result) {
 
   @Test
   public void groupByHavingTest() throws Exception {
-    JSONObject result = executeQuery(String.format(
-        "SELECT gender " +
-            "FROM %s " +
-            "GROUP BY gender " +
-            "HAVING COUNT(*) > 0", TEST_INDEX_ACCOUNT));
+    JSONObject result =
+        executeQuery(
+            String.format(
+                "SELECT gender " + "FROM %s " + "GROUP BY gender " + "HAVING COUNT(*) > 0",
```

Comment:
Since this has been moved to be on the same row rather than split onto multiple rows I believe that we can have this whole row be in one set of quotations rather than split into multiple with + signs between them

Reply:
Good catch, thanks, fixed.